site/docs/v1/tech/identity-providers/samlv2-idp-initiated/index.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: doc
+title: SAML v2 IdP Initiated Identity Provider
+description: Login using a SAML v2 IdP Initiated Identity Provider
+---
+
+== Overview
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+This FusionAuth Identity Provider allows you to configure FusionAuth to accept a SAML request initiated by a SAMLv2 identity provider. With this Identity Provider, FusionAuth is acting as the SAMLv2 service provider (or SP).
+
+[WARNING.warning]
+====
+Using IdP initiated SSO with this Identity Provider is not recommended and is inherently less secure than a service provider initiated login. 
+
+In general FusionAuth does not recommend you utilize this type of login due to the inherent risks of accepting an unsolicited authentication response from a third-party identity provider.  
+
+If you must enable this feature to support a legacy integration or because the user experience is of the highest concern, then proceed.
+====
+
+Typically, a SAML flow begins at a service provider such as FusionAuth. In a service provider initiated login, the SAML Authentication Response is returned to the service provider in response to an authentication request. FusionAuth can validate certain parameters and only accept a response for a request that FusionAuth initiated. 
+
+In contrast, during an identity provider initiated login, the third-party identity provider sends FusionAuth a SAML Authentication Response indicating a user has been authenticated. FusionAuth did not initiate this request so it is unexpected and FusionAuth does not have the full context. This limits what can be verified and requires FusionAuth to allow a cross site request from this third party. 
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+* <<Create a SAML v2 IdP Initiated Identity Provider>>
+* <<Integration Details>>
+* <<CORS Configuration>>
+
+We also provide specific examples for configuring SAML with some providers whose implementation requires unique configuration. If you'd like us to provide additional examples, please open a request on https://github.com/FusionAuth/fusionauth-site/issues[GitHub].
```

Comment:
consider
If you would like to see an example in addition to what has already been documented, please open a request on https://github.com/FusionAuth/fusionauth-site/issues[GitHub].